colosseum/declaration.py

```diff
@@ -45,10 +48,23 @@ def deleter(self):
 
 def validated_property(name, choices, initial):
     "Define a simple CSS property attribute."
-    initial = choices.validate(initial)
+    try:
+        initial = choices.validate(initial)
+    except ValueError:
+        try:
+            # The initial value might be a OtherProperty or Custom class with a value method
```

Comment:
It doesn't just need to exist - it also has to be a callable.

Reply:
Fixed!